packages/unlock-js/scripts/compressAbi.js

```diff
@@ -13,22 +13,24 @@ const path = require('path')
 /* eslint-disable import/no-extraneous-dependencies */
 const abis = require('@unlock-protocol/contracts')
 
-const unlockVersions = ['v4', 'v6', 'v7', 'v8', 'v9']
+const unlockVersions = ['v4', 'v6', 'v7', 'v8', 'v9', 'v10']
 const publicLockVersions = ['v4', 'v6', 'v7', 'v8']
```

Comment:
Hum, we should add tests for `publicLockVersions` `v9` as well, correct?

Reply:
actually we still don't have support for publicLock v9 in unlock-js, so thats another thing we should add

Reply:
added v9 to #8050